lib/pages/account_creation.dart

```diff
@@ -94,3 +107,22 @@ class _AccountCreationState extends State<AccountCreation> {
     );
   }
 }
+
+Future _buildErrorDialog(BuildContext context, _message) {
+  return showDialog(
+    builder: (context) {
+      return AlertDialog(
+        title: Text('Error Message'),
+        content: Text(_message),
+        actions: [
+          FlatButton(
+              child: Text('Cancel'),
+              onPressed: () {
```

Comment:
The `.pop()` action is not async? Because otherwise we need to add `async` after `onPressed:() ..`

Reply:
.pop() does not seem to be async. Added keyword